src/service-override/accessibility.ts

```diff
@@ -15,6 +19,8 @@ registerAssets(audioAssets)
 export default function getServiceOverride (): IEditorOverrideServices {
   return {
     [IAccessibleViewService.toString()]: new SyncDescriptor(AccessibleViewService, [], true),
-    [IAccessibilitySignalService.toString()]: new SyncDescriptor(AccessibilitySignalService, [], true)
+    [IAccessibilitySignalService.toString()]: new SyncDescriptor(AccessibilitySignalService, [], true),
+    [IAccessibilityService.toString()]: new SyncDescriptor(AccessibilityService, [], true),
```

Comment:
It seems superfluous but meh, it can't hurt I guess!

Reply:
Who knows if they'll add some logic in the future! 😂